SDWebImage/UIImage+GIF.m

```diff
@@ -33,11 +33,13 @@ + (UIImage *)sd_animatedGIFWithData:(NSData *)data {
         for (size_t i = 0; i < count; i++) {
             CGImageRef image = CGImageSourceCreateImageAtIndex(source, i, NULL);
 
```

Comment:
If you add a check for nil here, you can also protect against crashes when the frame is malformed.